### PR TITLE
Renamed package and updated existing functions

### LINKED ISSUE #9: Add lingua () function
For translating text to English.

### LINKED ISSUE #23: Develop code_agreements to differently weight various identifying information

### LINKED ISSUE #28: Create cheatsheet
https://www.rstudio.com/resources/cheatsheets/how-to-contribute-a-cheatsheet/

### LINKED ISSUE #47: Add a function to structure texts
Create a function to structure texts as lists of lists

### LINKED ISSUE #48: Fix export_data() by making it automatically detect the package name when exporting data
By changing the data preparation template.

### LINKED ISSUE #57: Update `code_agreements()` and `condense_qID()` to run more efficiently
Some tools as profiling might help to make the functions more concise and faster.

### LINKED ISSUE #58: Rename package, and all templates

### PR DESCRIPTION
**Pull request by jaeltan**
# Description
Minor changes to package including renaming the package and updating existing functions to work better with treaties and data.

## Package
* Closed #58 by renaming package from `qCreate` to `manypkgs`
* Closed #28 by adding cheatsheet to README

## Correction
* Closed #57 by updating `code agreements()` and `condense_qID()` to make code more concise
* Closed #48 by updating data preparation template in `export_data()`
* Updated the countryregex table used in `code_parties()` so that countries are detected more accurately

## Connection
* Updated `standardise_titles()`
  * Closed #9 by adding `lingua()` function for translating text to English
* Closed #47 by adding functions to structure treaty text
  * Added `split_treaty()` to split treaty text into preamble and articles
  * Added `get_treaty()` to get all matching articles by number or word match

# Checklist:

- [X] PR form
  - [X] I have given this pull request an informative title
  - [X] Description above itemizes changes under subtitles, e.g. "## Collection""
  - [X] Any closed, fixed, or related issues are referenced and explained in the description above, e.g. "Fixed #0 by adding A"
  - [X] Package builds on my OS without issues
- [x] PR checks all pass for latest commit
  - [x] Package builds on Mac
  - [x] Package builds on Windows
  - [x] Package builds on Linux
  - [x] CodeCov check: Package improves or maintains good test coverage
  - [x] CodeFactor check: Package improves or maintains good style
- [X] Documentation
  - [X] Any new or modified functions or data have roxygen style documentation in their .R scripts
  - [X] Any longer functions are commented inline so that it is easier to debug in the future
  - [X] PR description above and the NEWS.md file are aligned
  - [X] DESCRIPTION file version is bumped by the appropriate increment (major, minor, patch)

### CODE REVIEW

File: R/split_treaty.R [hunk 1]
```diff
@@ -0,0 +1,74 @@
+#' Split Treaty Texts
+#' 
+#' Split treaty texts into lists that reflect a structure
+#' based on having a preamble and several articles.
+#' @param textvar text variable
+#' @importFrom stringr str_detect
+#' @importFrom stringi stri_trans_general
+#' @return A structured list for each agreement
+#' @examples
+#' \dontrun{
```
Comment (henriquesposito) [inline R/split_treaty.R:10]: Change to donttest please

File: R/split_treaty.R [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+#' @param article Either the "preamble" or an article number
+#' across all treties to be returned.
+#' It can be left NULL if user intends to look for a word in
+#' all treaties.
+#' @param match A regex match for a word(s) or expression.
+#' For multiple words, please use "|" to divide them.
+#' @importFrom purrr map_chr
+#' @return A list of treaty sections of the same length
+#' @examples
+#' \dontrun{
```
Comment (henriquesposito) [inline R/split_treaty.R:46]: donttest

File: inst/templates/Package-DESC.dcf [hunk 3]
```diff
@@ -1,5 +1,5 @@
 Package: {{{package}}}
-Title: A Package for the qData Ecosystem
+Title: A Package for the many packages Ecosystem
```
Comment (henriquesposito) [inline inst/templates/Package-DESC.dcf:2]: universe

File: inst/templates/qData-prep.R [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # {{{dataset}}} Preparation Script
 
 # This is a template for importing, cleaning, and exporting data
-# ready for the qPackage.
+# ready for many packages.
```
Comment (henriquesposito) [inline inst/templates/qData-prep.R:4]: the many packages universe

File: inst/templates/test_actors.R [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# Test if {{{dataset}}} meets the q ecosystem requirements
+# Test if {{{dataset}}} meets the many packages ecosystem requirements
```
Comment (henriquesposito) [inline inst/templates/test_actors.R:1]: universe

File: inst/templates/test_agreements.R [hunk 6]
```diff
@@ -1,4 +1,4 @@
-# Test if {{{dataset}}} meets the q ecosystem requirements
+# Test if {{{dataset}}} meets the many packages ecosystem requirements
```
Comment (henriquesposito) [inline inst/templates/test_agreements.R:1]: universe

File: inst/templates/test_general.R [hunk 7]
```diff
@@ -1,4 +1,4 @@
-# Test if {{{dataset}}} meets the q ecosystem requirements
+# Test if {{{dataset}}} meets the many packages ecosystem requirements
```
Comment (henriquesposito) [inline inst/templates/test_general.R:1]: universe

File: inst/templates/test_memberships.R [hunk 8]
```diff
@@ -1,4 +1,4 @@
-# Test if {{{dataset}}} meets the q ecosystem requirements
+# Test if {{{dataset}}} meets the many packages ecosystem requirements
```
Comment (henriquesposito) [inline inst/templates/test_memberships.R:1]: universe

File: inst/templates/test_states.R [hunk 9]
```diff
@@ -1,4 +1,4 @@
-# Test if {{{dataset}}} meets the q ecosystem requirements
+# Test if {{{dataset}}} meets the many packages ecosystem requirements
```
Comment (henriquesposito) [inline inst/templates/test_states.R:1]: universe

File: pkgdown/_pkgdown.yml [hunk 10]
```diff
@@ -1,4 +1,4 @@
-url: https://globalgov.github.io/qCreate/
+url: https://globalgov.github.io/manypkgs/
```
Comment (henriquesposito) [inline pkgdown/_pkgdown.yml:1]: Can you just please run pkgdown::build_site() locally to make sure all is good to go with the website? Thank you.

**Review by henriquesposito — CHANGES_REQUESTED** [review]
Great job @EstherPeev and @jaeltan. thank you! There are a few more changes to go, but we are almost there! Please let me know if you have any questions about my comments

File: pkgdown/_pkgdown.yml (hunk 10 above)
Comment (EstherPeev) [reply pkgdown/_pkgdown.yml:1]: Done!

**Comment by EstherPeev** [comment]
While going over the files in manypkgs, I saw that there are many references to qIDs (i.e. in `code_agreements()`, `condense_qID()`, in the cheatsheet). 
@henriquesposito Should we also rename qID into treaty_ID? Or something else?

File: R/data_import.R [hunk 11]
```diff
@@ -130,27 +130,28 @@ import_data <- function(dataset = NULL,
...
-  usethis::ui_todo("Use {usethis::ui_code('qCreate::export_data()')} to add prepared data to package")
+  usethis::ui_todo("Use {usethis::ui_code('manypkgs::export_data()')} to add prepared data to package")
 
 }
 
 #' Add .bib file
 #'
-#' Add .bib file template to help cite the datasets in a qPackage
+#' Add .bib file template to help cite the datasets in a package from
+#' our ecosystem.
```
Comment (henriquesposito) [inline R/data_import.R:140]: universe

File: R/package_setup.R [hunk 12]
```diff
@@ -164,106 +164,106 @@ setup_package <- function(package = NULL,
...
-            data = list(package = package,
+  manytemplate("Package-README.Rmd",
+               "README.Rmd",
+               data = list(package = package,
                         author = author),
-            path = path)
+               path = path)
   usethis::ui_done("Added standard README.")
-  # TODO: Add badges to qPackage README
+  # TODO: Add badges to the Package README
```
[the remark this answers is not in the document]
Comment (henriquesposito) [reply R/package_setup.R:189]: Remove todo

File: inst/templates/Package-DESC.dcf [hunk 13]
```diff
@@ -1,5 +1,5 @@
 Package: {{{package}}}
-Title: A Package for the qData Ecosystem
+Title: A Package for the "many"" packages universe
```
Comment (henriquesposito) [inline inst/templates/Package-DESC.dcf:2]: remove one "

File: R/data_import.R [hunk 14]
```diff
@@ -116,8 +116,8 @@ import_data <- function(dataset = NULL,
     } else stop("File type not recognised")
 
   # Create preparation template
-  qtemplate(
-    "qData-prep.R",
+  manytemplate(
+    "package-prep.R",
```
Comment (henriquesposito) [inline R/data_import.R:120]: If most files start with an upper case  "P", please make sure all files follow for consistency.

File: inst/templates/test_memberships.R [hunk 15]
```diff
@@ -13,7 +13,7 @@ test_that("missing observations are reported correctly", {
 # Uniformity tests (agreements have a source ID, a string title, a signature and
 # entry into force date)
 test_that("datasets have the required variables", {
-  expect_col_exists({{{dab}}}[["{{{dat}}}"]], vars(Country))
+  expect_col_exists({{{dab}}}[["{{{dat}}}"]], vars(CountryID))
```
Comment (henriquesposito) [inline inst/templates/test_memberships.R:16]: Should it not be a column that either starts with or end or contain the word "ID"? CountryID sounds very specific...

**Review by henriquesposito — CHANGES_REQUESTED** [review]
Great job @EstherPeev , thank you! Almost there, just a few more minor changes to go and we should be good to go

**Pull request merged**